client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -96,10 +96,6 @@ export default class ReserveSubmitButton extends Vue {
 
     switch (setup) {
       case 'assumed':
-        if (this.$xproMapping['ASSUMED'].includes(entity_type_cd)) {
-          newReqModule.mutateRequestActionOriginal(request_action_cd)
-          newReqModule.mutateRequestAction('ASSUMED')
-        }
         newReqModule.mutateAssumedNameOriginal()
```

Comment:
Normally the mutations take a parameter. Is this a method disguised as a mutation?

Reply:
Yes